tests/Whoops/RunTest.php

```diff
@@ -402,6 +406,7 @@ public function testSilenceErrorsInPaths()
 
     /**
      * @covers Whoops\Run::handleError
+     * @requires PHP < 8
```

Comment:
Why does this not work on php8

Reply:
Because this is a different kind of error now.

Reply:
PHP does not permit silencing it.